Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    - templateName: objc-string-h<br />      output: Localizable.h</pre><br />    - templateName: objc-string-m<br />      output: Localizable.m</pre> |
```

Comment:
Extra `</pre>` in the middle of that block makes the rendering bad after the first `templateName`+`output`: the second `templateName`+`output` for the `.m` isn't rendered in monospaced nor properly. Removing that extra `</pre>` before the `<br />` should solve it.

Reply:
got it, thanks